sim/hunter/explosive_shot.go

```diff
@@ -62,7 +67,7 @@ func (hunter *Hunter) registerExplosiveShotSpell(timer *core.Timer) {
 			NumberOfTicks: 2,
 			TickLength:    time.Second * 1,
 			OnSnapshot: func(sim *core.Simulation, target *core.Unit, dot *core.Dot, isRollover bool) {
-				baseDamage := sim.Roll(baseLowDamage, baseHighDamage) + 0.039*dot.Spell.RangedAttackPower(target)
+				baseDamage := ( sim.Roll(baseLowDamage, baseHighDamage) + 0.039*dot.Spell.RangedAttackPower(target) ) * damageMult
```

Comment:
I think all of these multipliers in these spell files should be applied to the `DamageMultiplier` field since they're not base damage multipliers as far as I can tell. That's effectively what you're doing by multiplying each damage calculation like this

Reply:
Didn't realize we had that field available, oops! I'll apply the bonus there instead